roles/ovirt-engine-install-packages/meta/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+galaxy_info:
+  author: Tareq Alayan
+  description: Role to install packages required for deployin oVirt engine.
+  company: Red Hat, Inc.
+
+  license: Apache License 2.0
+
+  min_ansible_version: 2.3
+
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  - name: Fedora
```

Comment:
Let's not explicitly set fedora versions for all roles, so we don't need to maintain it

Reply:
ok